kyaml/runfn/runfn.go

```diff
@@ -303,6 +303,28 @@ func (r RunFns) mergeContainerEnv(envs []string) []string {
 	return declarative.Raw()
 }
 
+// mergeExecEnv will merge the envs specified by command line (imperative) and config
+// file (declarative). If they have same key, the imperative value will be respected.
```

Comment:
@xvzf if the same value is specified through command line and env var, would it make sense to error out instead?

I'm fine with any option that makes sense from the user perspective - as this is a one off case with exec functions. We would just need to explicitly specify docs on the stance we take.

cc: @koba1t @natasha41575

Reply:
Hey

It's the same behavior as for the container env merging (aka CLI flag takes precedence). Afaik it's the default behavior in most applications. Open to change/document it though!